RecoBTag/DeepFlavour/interface/sorting_modules.h

```diff
@@ -0,0 +1,187 @@
+/*
+ * sorting_modules.h
+ *
+ *  Created on: 5 Mar 2017
+ *      Author: jkiesele
+ */
+
+#ifndef DEEPNTUPLES_DEEPNTUPLIZER_INTERFACE_SORTING_MODULES_H_
+#define DEEPNTUPLES_DEEPNTUPLIZER_INTERFACE_SORTING_MODULES_H_
+
+#include <algorithm>
+#include <cmath>
+#include <vector>
+
+namespace sorting{
+
+/*
+ * the std::sort function needs a strict ordering.
+ * means if your_function(a,b) is true, than your_function(b,a) must be false for all a and b
+ */
+
+template<class T>
+bool comparePt(T a, T b){
```

Comment:
where is this used?

Reply:
not anymore, will remove

Reply:
removed with https://github.com/cms-sw/cmssw/pull/19893/commits/6ad48f2a3527cf49f3a753cf57bfe30380048472